### ISSUE TITLE
Release library as Nuget Package

### ISSUE THREAD

**Issue by Lexcess** [Microsoft/groove-api-sdk-csharp#2]
While it is great you have an SDK here it would be useful to have it released on Nuget.org as a package (ideally a .Net Standard one) so that updates can easily be consumed. If it is already available on Nuget perhaps you could put it up under releases.

**Comment by bfreydier** [comment]
Thanks for reporting this, it is in our backlog and should be online shortly.

**Comment by nojaf** [comment]
Will Mono be supported as well?

**Comment by pherbel** [comment]
Do you plan support .Net standard and uwp target?

**Comment by refre** [comment]
Too bad, it's not possible to use de PCL in Xamarin. Do you plan to add it?

**Comment by ThePyrobri** [comment]
Is this yet available on NuGet?

**Comment by villasv** [comment]
Apparently not yet, not even as prerelease.

**Comment by nojaf** [comment]
Guess we can close this after all :wink:

**Comment by Lexcess** [comment]
Yep. I guess I'm glad they never did it, otherwise I would have wasted a lot of time on a project that was destined to die.
